tools/mksurfdata_map/src/mkpftUtilsMod.F90

```diff
@@ -23,8 +23,9 @@ module mkpftUtilsMod
   !
   ! !PUBLIC MEMBER FUNCTIONS:
   !
-  public :: convert_from_p2g      ! Convert a p2g array into pct_pft_type objects
-  public :: adjust_total_veg_area ! Adjust the total vegetated area (natural veg & crop) to a new specified total
+  public :: convert_from_p2g         ! Convert a p2g array into pct_pft_type objects
+  public :: adjust_total_veg_area    ! Adjust the total vegetated area (natural veg & crop) to a new specified total
```

Comment:
- [ ] I think adjust_total_veg_area is now unused. If so, it can be removed entirely.

Reply:
Done